monkey/monkey_island/cc/ui/src/components/pages/RunServerPage.js

```diff
@@ -16,11 +16,8 @@ class RunServerPageComponent extends React.Component {
       <Col sm={{offset: 3, span: 9}} md={{offset: 3, span: 9}}
            lg={{offset: 3, span: 9}} xl={{offset: 2, span: 7}}
            className={'main'}>
-        <h1 className="page-title">Welcome to the Monkey Island Server</h1>
+        <h1 className="page-title">Getting Started</h1>
```

Comment:
We should rename this whole page to GetStarted page, this has nothing to do with running the server.